### ISSUE TITLE
Support for times already past in `chime-at`

### ISSUE THREAD

**Issue by luminusian** [jarohen/chime#19]
#7 doesn't seem to be resolved.

Sometimes the following code produces an error.

```
(let [start-time (t/now)]
  (chime/chime-at [(-> 1 t/millis t/from-now)] println {:start-time start-time}))
=>
#object[chime$chime_at$cancel_BANG___15477
        0x40b6d48d
        "chime$chime_at$cancel_BANG___15477@40b6d48d"]
Exception in thread "async-dispatch-2" java.lang.IllegalArgumentException: No implementation of method: :before? of protocol: #'clj-time.core/DateTimeProtocol found for class: nil
	at clojure.core$_cache_protocol_fn.invokeStatic(core_deftype.clj:568)
	at clojure.core$_cache_protocol_fn.invoke(core_deftype.clj:560)
	at clj_time.core$eval6044$fn__6089$G__6021__6096.invoke(core.clj:102)
	at chime$ms_between.invokeStatic(chime.clj:8)
	at chime$ms_between.invoke(chime.clj:7)
	at chime$chime_ch$fn__15317$state_machine__12479__auto____15332$fn__15334.invoke(chime.clj:38)
	at chime$chime_ch$fn__15317$state_machine__12479__auto____15332.invoke(chime.clj:38)
	at clojure.core.async.impl.ioc_macros$run_state_machine.invokeStatic(ioc_macros.clj:973)
	at clojure.core.async.impl.ioc_macros$run_state_machine.invoke(ioc_macros.clj:972)
	at clojure.core.async.impl.ioc_macros$run_state_machine_wrapped.invokeStatic(ioc_macros.clj:977)
	at clojure.core.async.impl.ioc_macros$run_state_machine_wrapped.invoke(ioc_macros.clj:975)
	at chime$chime_ch$fn__15317.invoke(chime.clj:38)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by jarohen** [comment]
Hi @luminusian - thanks for the report. Could you clarify what `{:start-time start-time}` is doing here? As far as I know it isn't a chime option!

James

**Comment by jarohen** [comment]
This might be related to #20 (fix released in Chime 0.2.1) - could you let me know if this fixes your issue?

**Comment by luminusian** [comment]
Thank you.
{:start-time start-time} is taken from the following comment.
https://github.com/jarohen/chime/issues/7#issuecomment-154860754

**Comment by jarohen** [comment]
fixed in 0.3.2
